third_party/move/move-model/src/lib.rs

```diff
@@ -307,11 +307,23 @@ pub fn run_model_builder_with_options_and_compilation_flags<
 
     // Extract the module/script closure
     let mut visited_modules = BTreeSet::new();
+    // Extract the module closure for the vector module
```

Comment:
"module dependency closure" would be more clear here (and on line 308), especially for those of us with "[function] closure" on the brain.